nifi-nar-bundles/nifi-snowflake-bundle/nifi-snowflake-services/src/main/java/org/apache/nifi/snowflake/service/StandardSnowflakeIngestManagerProviderService.java

```diff
@@ -0,0 +1,199 @@
+            .required(true)
+            .allowableValues(AccountIdentifierFormat.class)
+            .defaultValue(AccountIdentifierFormat.ACCOUNT_NAME.getValue())
+            .build();
+
+    public static final PropertyDescriptor HOST_URL = new PropertyDescriptor.Builder()
+            .name("host-url")
+            .displayName("Snowflake URL")
+            .description("Example host url: [account-locator].[cloud-region].[cloud].snowflakecomputing.com")
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
+            .required(true)
+            .dependsOn(ACCOUNT_IDENTIFIER_FORMAT, AccountIdentifierFormat.FULL_URL)
+            .build();
+
+    public static final PropertyDescriptor ACCOUNT_LOCATOR = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(CommonProperties.ACCOUNT_LOCATOR)
+            .dependsOn(ACCOUNT_IDENTIFIER_FORMAT, AccountIdentifierFormat.ACCOUNT_LOCATOR)
+            .build();
+
+    public static final PropertyDescriptor CLOUD_REGION = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(CommonProperties.CLOUD_REGION)
+            .dependsOn(ACCOUNT_IDENTIFIER_FORMAT, AccountIdentifierFormat.ACCOUNT_LOCATOR)
+            .build();
+
+    public static final PropertyDescriptor CLOUD_TYPE = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(CommonProperties.CLOUD_TYPE)
+            .dependsOn(ACCOUNT_IDENTIFIER_FORMAT, AccountIdentifierFormat.ACCOUNT_LOCATOR)
+            .build();
+
+    public static final PropertyDescriptor ORGANIZATION_NAME = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(CommonProperties.ORGANIZATION_NAME)
+            .dependsOn(ACCOUNT_IDENTIFIER_FORMAT, AccountIdentifierFormat.ACCOUNT_NAME)
+            .build();
+
+    public static final PropertyDescriptor ACCOUNT_NAME = new PropertyDescriptor.Builder()
+            .fromPropertyDescriptor(CommonProperties.ACCOUNT_NAME)
+            .dependsOn(ACCOUNT_IDENTIFIER_FORMAT, AccountIdentifierFormat.ACCOUNT_NAME)
+            .build();
+
+    public static final PropertyDescriptor USER_NAME = new PropertyDescriptor.Builder()
+            .name("user-name")
+            .displayName("User Name")
+            .description("The Snowflake user name.")
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
+            .required(true)
+            .build();
+
+    public static final PropertyDescriptor PIPE_NAME = new PropertyDescriptor.Builder()
+            .name("pipe-name")
+            .displayName("Pipe Name")
+            .description("The Snowflake pipe's name to ingest from.")
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
+            .required(true)
+            .build();
```

Comment:
I would consider splitting the property to `Database`, `Schema` and `Pipe`.
Otherwise, it should be mentioned in the description that the fully qualified name is expected containing the database and the schema. However, the 3 properties would be more straightforward in my opinion.